qn-7.js

```diff
@@ -0,0 +1,119 @@
+// number one
+const products = [
+  { name: "Product 1", count: 10, price: 100 },
+  { name: "Product 2", count: 5, price: 50 },
+  { name: "Product 3", count: 20, price: 200 },
+  { name: "Product 5", count: 0, price: 50 },
+
+];
+
+function getTotalItemCount() {
+  let totalCount = 0;
+  products.forEach((product) => {
+    totalCount += product.count;
+  });
+  console.log(totalCount);
+}
+getTotalItemCount();
+
+//   number two
+function createNewItem(name, count, price) {
+  const newItem = { name, count, price };
+  products.forEach(item => {
+  if(item.name !== newItem.name){
+  products.push(newItem);
+
+  }else{
+    console.log("the product already exist.");
+  }
+});
+ 
+  console.log("you added "+newItem.name +" sucsesfully" );
+}
+
+createNewItem("Product 4", 15, 150);
+//   number three
+function getAvailableItems() {
+  console.log(products.filter((product) => product.count > 0)); 
+}
+
+getAvailableItems();
+//   number four
+function enableItem(name) {
+  const item = products.find((product) => product.name === name);
+  if (item) {
+    item.count = 1;
```

Comment:
Why have you changed the count to 1?

Reply:
I thought maybe in this way we could disable and enable the product but then i saw in guys' projects declaring a boolean in each product was a better idea